docs/source/releases/v1_11_7a0.rst

```diff
@@ -32,6 +32,7 @@ Version 1.11.7a0 (2023-10-26)
 * Added unit test configuration file
```

Comment:
Opening conversation to remind us to NOT merge this until v1.12.0-release has been merged to main, and this destination branch is automatically updated to point to main.